packages/common-testnets/src/nodes/tenderly/TenderlyTestnetFactory.ts

```diff
@@ -7,48 +9,44 @@ import { CreateNetworkArgs, TestnetCreateResult, TestnetFactory } from '../../Te
 import { getTenderlyClient } from './TenderlyClient.js'
 
 export class TenderlyTestnetFactory implements TestnetFactory {
-  constructor(private readonly opts: { apiKey: string; account: string; project: string }) {}
+  constructor(
+    private readonly opts: { apiKey: string; account: string; project: string },
+    private readonly httpClient = new HttpClient(Logger.SILENT),
```

Comment:
```suggestion
    private readonly httpClient,
```
i think i prefer to have explicit log status here